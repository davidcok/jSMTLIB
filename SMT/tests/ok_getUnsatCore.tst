; get-unsat-core after sat/unknown
(set-option :produce-unsat-cores true)
(set-logic QF_UF)
(assert (! false :named F))
(check-sat)
(get-unsat-core)
