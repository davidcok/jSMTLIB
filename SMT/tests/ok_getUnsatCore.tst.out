unsupported
success
success
unsat
(get-unsat-core)
^^^^^^^^^^^^^^^^
(error "The get-unsat-core command is only valid if :produce-unsat-cores has been enabled")
